src/main/java/duke/ui/Ui.java

```diff
@@ -0,0 +1,99 @@
+package duke.ui;
+
+import java.util.ArrayList;
+
+import duke.task.Task;
+
+/**
+ * Handles Ui related functions such as input and output.
+ */
+public class Ui {
+    /**
+     * Message shown when user opens the application
+     */
+    public static final String WELCOME_MESSAGE = "Hello! I'm Chip the Squirrel! How can I help you today?";
+    /**
+     * Message shown when user closes the application
+     */
+    public static final String GOODBYE_MESSAGE = "Bye! Hope to see you again soon!";
+
+    private static String joinStringsWithNewlines(String... strings) {
```

Comment:
I like that you have a method to deal with string formatting. Maybe the method name should be joinStringWithNewLines instead? (ie Lines instead of line)